Ghidra/Features/Decompiler/src/decompile/cpp/translate.cc

```diff
@@ -896,13 +896,12 @@ void PcodeEmit::restoreXmlOp(const Element *el,const AddrSpaceManager *manage)
 { // Read a raw pcode op from DOM (and dump it)
   int4 opcode;
   VarnodeData outvar;
-  VarnodeData invar[30];
+  vector<VarnodeData> invar;
   VarnodeData *outptr;
 
   istringstream i(el->getAttributeValue("code"));
   i >> opcode;
-  const List &list(el->getChildren());
-  List::const_iterator iter = list.begin();
```

Comment:
We don't need to make a copy of a list of constant elements. Using its iterator is sufficient.